es/src/it/scala/monix/connect/es/ElasticsearchSinkSuite.scala

```diff
@@ -0,0 +1,30 @@
+package monix.connect.es
+
+import monix.execution.Scheduler.Implicits.global
+import monix.reactive.Observable
+import org.scalacheck.Gen
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+
+class ElasticsearchSinkSuite extends AnyFlatSpecLike with Fixture with Matchers with BeforeAndAfterEach {
+  import com.sksamuel.elastic4s.http.ElasticDsl._
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    elasticClient.execute(deleteIndex("*")).runSyncUnsafe()
+  }
+  "ElasticsearchSink" should "update a element in batches" in {
+    val updateRequests = Gen.listOfN(10, genUpdateRequest).sample.get
```

Comment:
could we separate each the test in the three steps (as it is done in other connectors)

//given
(values perepared for the test)

//when 
(actions)

//then
(checks)

Reply:
ok